src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -26,6 +26,27 @@
 #include "tvgCommon.h"
 #include "tvgArray.h"
 
+struct Box
```

Comment:
Maybe we can reuse this:

`struct RenderRegion  // tvgRender.h(83)`

or maybe with the refactoring `template<typename T = int32_t> Geometry //tvgMath.h`

Reply:
after templating the RenderRegion the binary file  increased by 4k

Reply:
@mgrudzinska got it, let's forget it.